app/Providers/AppServiceProvider.php

```diff
@@ -14,7 +17,13 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        $this->app->bind(RequestActivityLoggerInterface::class, function () {
+            if (App::environment('production')) {
+                return $this->app->make(RequestActivityLoggerInterface::class);
```

Comment:
ProductionRequestActivityLogger also still not used.

Reply:
yes my mistake